week-2-wide-perspective/w02e02-data-manipulation/package.json

```diff
@@ -1,5 +1,10 @@
 {
   "name": "w02e02-data-manipulation",
   "version": "1.0.0",
-  "dependencies": {}
+  "dependencies": {
```

Comment:
brakuje tutaj skryptu do uruchomienia `back-end` - super by było żeby nie trzeba było się zastanawiać jak to uruchomić :)

Wiem że to opisałeś w `PR` ale zobacz, że:
```json
"backend": "json-server --watch db/db.json"
```
robi robotę